### PR TITLE
addeed single video page

### PR DESCRIPTION
**Pull request by Developers-codz**
## Files To Check 
`src/components/aside/Aside.jsx`
`src/page/eachvideo/Video.jsx`
`src/page/home/Home.jsx`

## [Live](https://deploy-preview-3--play-arts.netlify.app/)

### CODE REVIEW

File: src/context/video-context.js [hunk 1]
```diff
@@ -1,15 +1,33 @@
...
 const VideoContext = createContext();
 
 const VideoProvider = ({ children }) => {
+  const [videos, setVideos] = useState([]);
+  useEffect(async () => {
+    try {
+      const response = await axios.get("/api/videos");
+      setVideos(response.data.videos);
+    } catch (err) {
+      console.log(err);
```
Comment (ApurvaSawant11) [inline src/context/video-context.js]: You can use `console.error`

**Review by ApurvaSawant11 — APPROVED** [review]
Looks good to me!

For UI, the border breaks in the middle for single video page. You can give that border to video container instead.

You can fix this later 
 
image

**Review by arjun-computer-geek — COMMENTED** [review]
Screenshot_2022-03-29_04-30-58

pills looking ugly

**Review by arjun-computer-geek — APPROVED** [review]
Else looks good

**Comment by Developers-codz** [comment]
> Screenshot_2022-03-29_04-30-58
> 
> pills looking ugly

for me its looking super good. See

Capture

File: src/context/video-context.js (hunk 1 above)
Comment (Developers-codz) [reply src/context/video-context.js]: sure

**Comment by Developers-codz** [comment]
> Looks good to me!
> 
> For UI, the border breaks in the middle for single video page. You can give that border to video container instead.
> 
> You can fix this later
> 
> image

that's because I have provided height to aside. Will fix that later

**Pull request merged**